src/Agent.Worker/CodeCoverage/CoberturaSummaryReader.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.TeamFoundation.TestManagement.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.CodeCoverage
```

Comment:
No. this reads from summary xml. not related to pom file. Enable codecoverage command does editing of POM xml.